### ISSUE TITLE
Product feed visibility incorrect value 

### ISSUE THREAD

**Issue by budzanowski** [woocommerce/facebook-for-woocommerce#2009]
It looks like the feed is sending staging value in the visibility fieled:
![image](https://user-images.githubusercontent.com/17271089/121359739-130c7d00-c934-11eb-8bbf-6ae4d29a344d.png)

Only `published` and hidden `values` are supported:
![image](https://user-images.githubusercontent.com/17271089/121359818-27507a00-c934-11eb-8ad9-e0ccf3dac4a9.png)

**Comment by haszari** [comment]
Thanks for finding this! Does this affect the new feed engine or all feed code – can you clarify that on the issue please (or if unsure someone can investigate).

Also - do we know what the impact of the `staging` value is – i.e. does this cause feed errors, or products shown in Facebook catalog that should be hidden (or both)?

**Comment by budzanowski** [comment]
I think that the products were missing from the catalog if they have been set to staging.

**Comment by haszari** [comment]
> I think that the products were missing from the catalog if they have been set to staging.

So to confirm – this was unlikely to be impacting stores?

**Comment by budzanowski** [comment]
It looks like it.

**Comment by csmcneill** [comment]
Encountered an issue with this on an AT site today. See 33720821-hc / 4734040-zen

User has some products that are not visible after syncing. FB logs indicate that it's because their visibility is set to `Staging`:

```
product\/baitlounge-baserange-wafter-complete-fish\/?attribute_diametre=24+mm","brand":"ADN CARPE","price":"9.9 EUR","availability":"in stock","visibility":"staging"
```

Other products that are visible have their visibility set to `Published`:

```
/product\/gardner-pva-system-3-tailles\/?attribute_taille=Recharge+Standard+25mm+7m","brand":"ADN CARPE","price":"7.07 EUR","availability":"in stock","visibility":"published"
```

I see that #2011 was merged, but it also doesn't seem to be released—the changelogs don't indicate that it was included. Is it slated to be included in 2.7.0?
